lib/diagnostic.rb

```diff
@@ -108,10 +108,10 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = 'person = {:favorite_number=> 11, :first_name => "Jim"}'
```

Comment:
This would work but remember we can shorten it even more:

`'person = {favorite_number: 11, first_name: "Jim"}'`